### PR TITLE
Add request model

### PR DESCRIPTION
**Pull request by sigmen**

### CODE REVIEW

File: lib/soap/request/options.ex [hunk 1]
```diff
@@ -0,0 +1,59 @@
+defmodule Soap.Request.Options do
```
Comment (Nitrino) [inline lib/soap/request/options.ex]: We do not need a structure for the request body and headers.
These parameters are transmitted once and no longer used

File: lib/soap/request.ex [hunk 2]
```diff
@@ -0,0 +1,20 @@
+defmodule Soap.Request do
+  defstruct [:method, :url, :body, :headers, :options]
+  @moduledoc """
+  Documentation for Soap.Request.
+  """
+
```
Comment (Nitrino) [inline lib/soap/request.ex]: Add specs for all functions please

File: lib/soap/request.ex [hunk 3]
```diff
@@ -0,0 +1,20 @@
+defmodule Soap.Request do
+  defstruct [:method, :url, :body, :headers, :options]
+  @moduledoc """
+  Documentation for Soap.Request.
```
Comment (Nitrino) [inline lib/soap/request.ex]: Let's write the documentation for models and methods right away

File: lib/soap/globals.ex [hunk 4]
```diff
@@ -0,0 +1,26 @@
+defmodule Soap.Globals do
+  defstruct [:headers, :body, :cookies]
+
+  @moduledoc """
```
Comment (Nitrino) [inline lib/soap/globals.ex]: The documentation should be on the first line in the module

File: lib/soap/request.ex [hunk 5]
```diff
@@ -0,0 +1,20 @@
+defmodule Soap.Request do
+  defstruct [:method, :url, :body, :headers, :options]
+  @moduledoc """
```
Comment (Nitrino) [inline lib/soap/request.ex]: The documentation should be on the first line in the module

Comment (sigmen) [reply lib/soap/request.ex]: Done.

File: lib/soap/request.ex (hunk 3 above)
Comment (sigmen) [reply lib/soap/request.ex]: Done.

File: lib/soap/request.ex (hunk 2 above)
Comment (sigmen) [reply lib/soap/request.ex]: Done.

File: lib/soap/request/options.ex (hunk 1 above)
Comment (sigmen) [reply lib/soap/request/options.ex]: Done.

**Pull request merged**